### ISSUE TITLE
"Anonymous AMD module used" error. Is this because I'm not using Grunt?

### ISSUE THREAD

**Issue by ssjjcc** [GoogleChrome/accessibility-developer-tools#293]
It me again! So I'm currently running my this Accessibility Audit on non-angular and angular webpages, I'm noticing a commonly occurring error. On angular pages, I get the same error message and the test terminates: `Failure during teardown: unknown error: Anonymous AMD module used, but it was not included as a dependency. This is most often caused by an anonymous define() from a script tag.` 

I believe this is occurring because I am not using Grunt and PhantomJS to do my testing, which, as I understand it, puts all HTML source code into standard HTML tags. (I was checking here: http://stackoverflow.com/questions/16524747/capturing-page-source-html-snapshot-after-angular-templating ). The only reason I haven't used Grunt and PhantomJS before is because the guys I'm developing with are trying to keep a standard testing suite among one another.

So my question is, am I seeing this error merely because I'm not using Grunt and PhantomJS to take an HTML snapshot of my pages, or have others seen and fixed this error without using Grunt and PhantomJS?
